src/themes/ec-core/_index.scss

```diff
@@ -1,7 +1,55 @@
 @use '@ecl/theme-dev/globals' with (
   $border-radius: 'no'
 );
-@use '@ecl/theme-dev/maps/color';
+@use '@ecl/theme-dev/maps/color' with (
+  $color: (
+    'blue': #004494,
+    'yellow': #ffd617,
```

Comment:
In the ec-core, there won't be overrides `with` if we agree to accept the `ec` as defaults.